docker/freertos-cheri/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Use latest Ubuntu as a base
+FROM ubuntu:20.04
+
+# Updates and dependencies
+RUN apt-get update
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=USA/Pacific
+RUN apt-get upgrade -y
+RUN apt-get install -y autoconf automake libtool pkg-config clang bison cmake ninja-build samba flex texinfo libglib2.0-dev libpixman-1-dev libarchive-dev libarchive-tools libbz2-dev libattr1-dev libcap-ng-dev wget git
+RUN ln -s /usr/bin/python3 /usr/bin/python
+RUN useradd -ms /bin/bash galoisuser
+
+# Set up for cloning
+RUN mkdir -p /opt/cheribuild
+RUN chown galoisuser /opt/cheribuild
+USER galoisuser
```

Comment:
Can we make it `besspinuser` for consistency among the docker images?

Reply:
Good point, fixed.